{{ .ProjectSnake }}/BUILD.bazel

```diff
@@ -50,12 +49,6 @@ js_library(
 # aspect:map_kind py_binary py_binary @aspect_rules_py//py:defs.bzl
 # aspect:map_kind py_library py_library @aspect_rules_py//py:defs.bzl
 # aspect:map_kind py_test py_test @aspect_rules_py//py:defs.bzl
-compile_pip_requirements(
-    name = "requirements",
-    requirements_in = "pyproject.toml",
-    requirements_txt = "requirements_lock.txt",
-)
-
 # Fetches metadata for python packages we depend on.
```

Comment:
@alexeagle, were these changes missing
```diff
+exports_files(["pyproject.toml"])
```
?

I just tried `aspect init` for the first time since this PR was merged. Immediately running `./repin.sh` (making no changes to what was generated) resulted in
```
ERROR: /private/var/folders/_t/xpfjyxwx727_zrtwcrhyfyjm0000gn/T/tmp.uBVmFZpqkL/foo/BUILD.bazel: no such target '//:pyproject.toml': target 'pyproject.toml' not declared in package '' defined by /private/var/folders/_t/xpfjyxwx727_zrtwcrhyfyjm0000gn/T/tmp.uBVmFZpqkL/foo/BUILD.bazel; however, a source file of this name exists.  (Perhaps add 'exports_files(["pyproject.toml"])' to /BUILD?)
```

Reply:
Ugh, some missing tests then. PR should have been red. I'll try to add this week